### PR TITLE
Port System.Text triple slash to docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported with DocsPortingTool

### CODE REVIEW

File: xml/System.Text/Rune.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@ CountLettersInString("Hello")
 But it won't work correctly for languages outside the Basic Multilingual Plane, such as Osage:
 
 ```csharp
-CountLettersInString("𐓏𐓘𐓻𐓘𐓻𐓟 𐒻𐓟")
+CountLettersInString("???????????? ????")
```
Comment (carlossanlop) [inline xml/System.Text/Rune.xml:110]:
```suggestion
CountLettersInString("𐓏𐓘𐓻𐓘𐓻𐓟 𐒻𐓟")
```

File: xml/System.Text/Rune.xml [hunk 2]
```diff
@@ -124,7 +124,7 @@ Here's equivalent code that works with a `ReadOnlySpan<char>`:
 The preceding code counts Osage letters correctly:
 
 ```csharp
-CountLettersInString("𐓏𐓘𐓻𐓘𐓻𐓟 𐒻𐓟")
+CountLettersInString("???????????? ????")
```
Comment (carlossanlop) [inline xml/System.Text/Rune.xml:127]:
```suggestion
CountLettersInString("𐓏𐓘𐓻𐓘𐓻𐓟 𐒻𐓟")
```

**Review by carlossanlop — COMMENTED** [review]
Revert non-ascii unexpected changes

File: xml/System.Text/Rune.xml [hunk 3]
```diff
@@ -1790,10 +1790,14 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="formatProvider" Type="System.IFormatProvider" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <param name="formatProvider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="format">The format to use.   
+ -or-   
+ A <see langword="null" /> reference (<see langword="Nothing" /> in Visual Basic) to use the default format defined for the type of the <see cref="T:System.IFormattable" /> implementation.</param>
```
Comment (gewarren) [inline xml/System.Text/Rune.xml:1795]:
```suggestion
<see langword="null" /> (<see langword="Nothing" /> in Visual Basic) to use the default format defined for the type of the <see cref="T:System.IFormattable" /> implementation.</param>
```

File: xml/System.Text/Rune.xml [hunk 4]
```diff
@@ -1790,10 +1790,14 @@ This member is an explicit interface member implementation. It can be used only
...
+        <param name="formatProvider">The provider to use to format the value.   
+ -or-   
+ A <see langword="null" /> reference (<see langword="Nothing" /> in Visual Basic) to obtain the numeric format information from the current locale setting of the operating system.</param>
```
Comment (gewarren) [inline xml/System.Text/Rune.xml:1798]:
```suggestion
<see langword="null" /> (<see langword="Nothing" /> in Visual Basic) to obtain the numeric format information from the current locale setting of the operating system.</param>
```

File: xml/System.Text/Rune.xml [hunk 5]
```diff
@@ -1822,12 +1826,13 @@ This member is an explicit interface member implementation. It can be used only
         <Parameter Name="provider" Type="System.IFormatProvider" Index="3" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="provider">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, this instance's value formatted as a span of characters.</param>
```
Comment (gewarren) [inline xml/System.Text/Rune.xml:1829]:
```suggestion
        <param name="destination">When this method returns, contains this instance's value formatted as a span of characters.</param>
```

File: xml/System.Text/Rune.xml [hunk 6]
```diff
@@ -1822,12 +1826,13 @@ This member is an explicit interface member implementation. It can be used only
...
+        <param name="charsWritten">When this method returns, the number of characters that were written in <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Text/Rune.xml:1830]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of characters that were written in <paramref name="destination" />.</param>
```

File: xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml [hunk 7]
```diff
@@ -151,10 +167,10 @@
         <Parameter Name="format" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="alignment">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to write.</param>
+        <param name="alignment">Minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml:171]:
```suggestion
        <param name="alignment">The minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```

File: xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml [hunk 8]
```diff
@@ -178,10 +194,10 @@
         <Parameter Name="format" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="alignment">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The span to write.</param>
+        <param name="alignment">Minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml:198]:
```suggestion
        <param name="alignment">The minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```

File: xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml [hunk 9]
```diff
@@ -205,10 +221,10 @@
         <Parameter Name="format" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="alignment">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to write.</param>
+        <param name="alignment">Minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml:225]:
```suggestion
        <param name="alignment">The minimum number of characters that should be written for this value. If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```

File: xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml [hunk 10]
```diff
@@ -264,9 +280,9 @@
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="value">To be added.</param>
-        <param name="alignment">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to write.</param>
+        <param name="alignment">Minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml:284]:
```suggestion
        <param name="alignment">The minimum number of characters that should be written for this value. If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```

File: xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml [hunk 11]
```diff
@@ -325,10 +341,10 @@
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="value">To be added.</param>
-        <param name="alignment">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value to write.</param>
+        <param name="alignment">Minimum number of characters that should be written for this value.  If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder+AppendInterpolatedStringHandler.xml:345]:
```suggestion
        <param name="alignment">The minimum number of characters that should be written for this value. If the value is negative, it indicates left-aligned and the required minimum is the absolute value.</param>
```

File: xml/System.Text/StringBuilder.xml [hunk 12]
```diff
@@ -2137,10 +2137,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="provider">To be added.</param>
-        <param name="handler">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="handler">The interpolated string to append.</param>
+        <summary>Appends the specified interpolated string to this instance.</summary>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder.xml:2142]:
```suggestion
        <summary>Appends the specified interpolated string to this instance using the specified format.</summary>
```

File: xml/System.Text/StringBuilder.xml [hunk 13]
```diff
@@ -3789,10 +3789,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="provider">To be added.</param>
-        <param name="handler">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="handler">The interpolated string to append.</param>
+        <summary>Appends the specified interpolated string followed by the default line terminator to the end of the current StringBuilder object.</summary>
```
Comment (gewarren) [inline xml/System.Text/StringBuilder.xml:3794]:
```suggestion
        <summary>Appends the specified interpolated string using the specified format, followed by the default line terminator, to the end of the current StringBuilder object.</summary>
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**